src/System.Reflection.Context/src/Configurations.props

```diff
@@ -2,8 +2,10 @@
 <Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <PropertyGroup>
     <BuildConfigurations>
-      netstandard;
```

Comment:
We should keep the netstandard configuration, as it helps prune the package graph for newer platforms.